code/game/objects/effects/landmarks/landmarks.dm

```diff
@@ -216,23 +216,25 @@
 
 /obj/effect/landmark/yautja_teleport
 	name = "yautja_teleport"
+	/// The index we registered as in mainship_yautja_desc or yautja_teleport_descs
+	var/desc_index
 
 /obj/effect/landmark/yautja_teleport/Initialize(mapload, ...)
 	. = ..()
 	var/turf/T = get_turf(src)
+	desc_index = T.loc.name + T.loc_to_string()
```

Comment:
```suggestion
	var/turf/turf = get_turf(src)
	desc_index = turf.loc.name + turf.loc_to_string()
```
See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#variables

Reply:
Honestly, I was given the code by a proper coder who said it might fix the issue, I don't actually know what you want me to do here since I can't really understand the code.

Reply:
Oh, its a suggested change, ok then.